views/random-beer.hbs

```diff
@@ -0,0 +1,11 @@
+{{#each beer}}
+{{> beerpartial}}
```

Comment:
I would use camelCase here as it's more aligned with convention than having a 2 words file all lower case